resources/public/api/conf/common/overview.md

```diff
@@ -1,12 +1,6 @@
-This API allows developers to retrieve, amend and delete expenses for:
+Use the Individuals Expenses API to submit, edit and retrieve information about a customer’s work-related expenses eligible for tax relief.
 
-- income for trade union and patent royalties
-- existing employment expenses
+This API is related to the following customer journey:
+* Submit employment expenses
 
-This API also allows developers to ignore HMRC provided employment expenses for a user.
-
-For information on how to connect to this API [see the Income Tax MTD end-to-end service guide](https://developer.service.hmrc.gov.uk/guides/income-tax-mtd-end-to-end-service-guide/).
-
-### Mapping APIs to Self Assessment tax return forms
-
-Some parameters in this API map to box numbers in one or more Self Assessment tax return forms. To learn more and to download this mapping information as a CSV file, go to [income-tax-mtd-changelog/mapping](https://github.com/hmrc/income-tax-mtd-changelog?tab=readme-ov-file#mapping-apis-to-self-assessment-tax-return-forms) on GitHub.
+Some parameters in this API map to box numbers in one or more Self Assessment tax return forms. To learn more and to download this mapping information as a CSV file, go to [income-tax-mtd-changelog/mapping-csv-files.md](https://github.com/hmrc/income-tax-mtd-changelog/blob/main/mapping/mapping-csv-files.md) GitHub. For information on how to connect to this API [see the Income Tax MTD end-to-end service guide](https://developer.service.hmrc.gov.uk/guides/income-tax-mtd-end-to-end-service-guide/).
```

Comment:
Can this please be broken down into three lines for the three sentences. Difficult to track as it is on one line so was difficult to spot that `on`  was missing before `GitHub`.